content/guides/user-guides/delfin/dashboard/_index.md

```diff
@@ -0,0 +1,170 @@
+![Delfin User Guide - 5](delfin-user-guide-5.png)
+
+![Delfin User Guide - 8](delfin-user-guide-8.png)
+![Delfin User Guide - 9](delfin-user-guide-9.png)
+![Delfin User Guide - 10](delfin-user-guide-10.png)
+![Delfin User Guide - 11](delfin-user-guide-11.png)
+![Delfin User Guide - 12](delfin-user-guide-12.png)
+![Delfin User Guide - 13](delfin-user-guide-13.png)
+![Delfin User Guide - 14](delfin-user-guide-14.png)
+![Delfin User Guide - 15](delfin-user-guide-15.png)
+
+
+### Storage Device Actions
+##### [\[Back to top\]](#delfin-features)  
+
+All the actions that can be performed by the user on storage devices
+Actions are available in the right-click menu in tree view and by clicking on "More" under the operations column in the list view table.
+1. Update access Info
+2. Configure Alert Source
+3. Remove Alert Source
+4. Sync Storage Device
+5. Configure Metric collection
+6. Delete Storage Device
+
+![Delfin User Guide - 16](delfin-user-guide-16.png)
+![Delfin User Guide - 17](delfin-user-guide-17.png)
+![Delfin User Guide - 18](delfin-user-guide-18.png)
+![Delfin User Guide - 19](delfin-user-guide-19.png)
+![Delfin User Guide - 20](delfin-user-guide-20.png)
+![Delfin User Guide - 21](delfin-user-guide-21.png)
+![Delfin User Guide - 22](delfin-user-guide-22.png)
+![Delfin User Guide - 23](delfin-user-guide-23.png)
+![Delfin User Guide - 24](delfin-user-guide-24.png)
+![Delfin User Guide - 25](delfin-user-guide-25.png)
+
+
+
+### Storage Device Details
+##### [\[Back to top\]](#delfin-features)  
+
+This is where you view the storage device details
+1. Configuration :  Basic details
+    - Volumes
+    - Storage Pools
+    - Controllers
+    - Ports
+    - Disks
+2. Capacity: Capacity summary for the storage device
+3. Alerts: All generated alerts for the storage device if alert source is specified.
+4. Performance Monitoring: Visualization of metrics using graphs if performance monitoring is enabled.
+![Delfin User Guide - 26](delfin-user-guide-26.png)
+![Delfin User Guide - 27](delfin-user-guide-27.png)
+![Delfin User Guide - 28](delfin-user-guide-28.png)
+![Delfin User Guide - 29](delfin-user-guide-29.png)
+![Delfin User Guide - 30](delfin-user-guide-30.png)
+![Delfin User Guide - 31](delfin-user-guide-31.png)
+![Delfin User Guide - 32](delfin-user-guide-32.png)
+![Delfin User Guide - 33](delfin-user-guide-33.png)
+![Delfin User Guide - 34](delfin-user-guide-34.png)
+![Delfin User Guide - 35](delfin-user-guide-35.png)
+![Delfin User Guide - 36](delfin-user-guide-36.png)
+
+
+### Performance Monitor
+##### [\[Back to top\]](#delfin-features)  
+
+This is where you view the performance monitoring information across the storage devices
+![Delfin User Guide - 37](delfin-user-guide-37.png)
+![Delfin User Guide - 38](delfin-user-guide-38.png)
+![Delfin User Guide - 39](delfin-user-guide-39.png)
+
+
+### Alertmanager
+##### [\[Back to top\]](#delfin-features)  
+
+This button launches the Alertmanager UI.
```

Comment:
Looks like u wanted to add some pic, is there any pic missed to attach here

Reply:
Yes. I think I missed one screenshot.

Reply:
Changed the content to reflect already available screenshots.